eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/RouteBlinding.scala

```diff
@@ -118,11 +119,10 @@ object RouteBlindingEncryptedDataCodecs {
   private val outgoingNodeId: Codec[OutgoingNodeId] = (("length" | constant(hex"21")) :: ("node_id" | publicKey)).as[OutgoingNodeId]
   private val pathId: Codec[PathId] = variableSizeBytesLong(varintoverflow, "path_id" | bytes).as[PathId]
   private val nextBlinding: Codec[NextBlinding] = (("length" | constant(hex"21")) :: ("blinding" | publicKey)).as[NextBlinding]
-  private val paymentRelay: Codec[PaymentRelay] = (
-    ("length" | constant(hex"0a")) ::
-      ("fee_base_msat" | millisatoshi32) ::
+  private val paymentRelay: Codec[PaymentRelay] = variableSizeBytesLong(varintoverflow,
+    ("cltv_expiry_delta" | cltvExpiryDelta) ::
       ("fee_proportional_millionths" | uint32) ::
-      ("cltv_expiry_delta" | cltvExpiryDelta)).as[PaymentRelay]
+      ("fee_base_msat" | tu32.xmap[MilliSatoshi](l => MilliSatoshi(l), m => m.toLong))).as[PaymentRelay]
```

Comment:
nit: we have a `tmillisatoshi` codec in `TlvCodecs.scala`, it's probably worth adding a `tmillisatoshi32` codec there?

Reply:
Done